integration_test/test_videoroom/test/integration/simulcast_test.exs

```diff
@@ -0,0 +1,323 @@
+defmodule TestVideoroom.Integration.SimulcastTest do
+  use TestVideoroomWeb.ConnCase, async: false
+
+  # in miliseconds
+  @peer_delay 500
+  # in miliseconds
+  @peer_duration 60_000
+  @room_url "http://localhost:4001"
+
+  # in miliseconds
+  @join_interval 20_000
+
+  @start_with_simulcast "start-simulcast"
+  @change_own_low "simulcast-local-low-encoding"
+  @change_own_medium "simulcast-local-medium-encoding"
+  @change_own_high "simulcast-local-high-encoding"
+  @set_peer_encoding_low "simulcast-peer-low-encoding"
+  @set_peer_encoding_medium "simulcast-peer-medium-encoding"
+  @simulcast_inbound_stats "simulcast-inbound-stats"
+  @simulcast_outbound_stats "simulcast-outbound-stats"
+  @browser_options %{count: 1, delay: @peer_delay, headless: true}
+  @test_duration 240_000
+
+  @tag timeout: @test_duration
+  test "User disable and then enable medium encoding" do
```

Comment:
Let's refactor to e.g. `test if disabling and enabling medium encoding again works correctly`